src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1662,6 +1663,39 @@ GUI_TEST_CLASS_DEFINITION(test_7451) {
     GTUtilsStartPage::checkRecentListUrl(os, "test_7451.fa", false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7454) {
+    // Open data/samples/PDB/1CF7.PDB.
+    // Increase the width of the Project View.
+    // Open any tab in the Options Panel and increase its width.
+    //     Expected: the current Sequence View is narrow.
+    // Find the action toolbar extension for the first sequence in the Sequence View (">>" button). Press ">>"->
+    //         "X Remove sequence".
+    //     Expected: no crash.
+    GTUtilsProject::openFile(os, dataDir + "samples/PDB/1CF7.PDB");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    QPoint splitterCenter = [&os] {
```

Comment:
Replace the following 5 lines with 2 lines line
```
QRect rect = GTWidget::findWidget(os, "project_view")->geometry();
QPoint splitterCenter = projectView->mapToGlobal({rect.right() + 4, rect.center().y()});
```

Reply:
❓

Reply:
You are using 'heavy artillery' for the trivial code: 2 lines or the trivial code looks more reasonable (and readable) than 5 lines of the modern one. 

The modern code is very good when it solves some problem: reduces boilerplate, improves readability.

Here the modern code only improves the 'visibility scope' which is not problem at all if you have only 2-liner like in my example.

Reply:
My fault: I didn't clarify the question.
Written strings are not compiled. Choose your preferred option:
```
QRect rect = GTWidget::findWidget(os, "project_view")->geometry();
QPoint splitterCenter = GTWidget::findWidget(os, "project_view")->mapToGlobal({rect.right() + 4, rect.center().y()});
```
or
```
QWidget* projectView = GTWidget::findWidget(os, "project_view");
QRect rect = projectView->geometry();
QPoint splitterCenter = projectView->mapToGlobal({rect.right() + 4, rect.center().y()});
```

Reply:
The first one is better because it is shorter and 
1) findWidget will never return nullptr
2) you will rarely debug this code because debugging of UI tests may 1) break your OS (like some button is pressed and have not been released yet) or 2) will switch the active window so the test can't continue.

Reply:
✓